esphome/components/alarm_control_panel/__init__.py

```diff
@@ -189,9 +189,11 @@ async def setup_alarm_control_panel_core_(var, config):
     for conf in config.get(CONF_ON_READY, []):
         trigger = cg.new_Pvariable(conf[CONF_TRIGGER_ID], var)
         await automation.build_automation(trigger, [], conf)
-    if (webserver_id := config.get(CONF_WEB_SERVER_ID)) is not None:
+    if (web_server_config := config.get("web_server")) is not None and (
```

Comment:
These should be constants
```suggestion
    if (web_server_config := config.get(CONF_WEB_SERVER)) is not None and (
```